packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -316,14 +327,18 @@ const UsersProfileFn = ({terms, slug, classes}: {
   const isEAForum = forumTypeSetting.get() === 'EAForum'
 
   const render = () => {
-    const user = getUserFromResults(results)
-    const { SunshineNewUsersProfileInfo, SingleColumnSection, SectionTitle, SequencesNewButton, LocalGroupsList, PostsListSettings, PostsList2, NewConversationButton, TagEditsByUser, NotifyMeButton, DialogGroup, SectionButton, SettingsButton, ContentItemBody, Loading, Error404, PermanentRedirect, HeadTags, Typography, ContentStyles } = Components
+    const { SunshineNewUsersProfileInfo, SingleColumnSection, SectionTitle, SequencesNewButton, LocalGroupsList,
+      PostsListSettings, PostsList2, NewConversationButton, TagEditsByUser, NotifyMeButton, DialogGroup,
+      SectionButton, SettingsButton, ContentItemBody, Loading, Error404, PermanentRedirect, HeadTags,
+      Typography, ContentStyles } = Components
+
```

Comment:
This code is the same, just cleaned up a bit. I moved the top line down under here.